core/src/test/java/org/wso2/carbon/kernel/internal/transports/DummyTransport.java

```diff
@@ -0,0 +1,50 @@
+package org.wso2.carbon.kernel.internal.transports;
+
```

Comment:
License header is already there. It was done after this PR was merged.
https://github.com/wso2/carbon-kernel/pull/693